src/Models/MessageRequest.cs

```diff
@@ -6,6 +6,7 @@ namespace StreamChat.Models
     public class MessageRequest : CustomDataBase
     {
         public string Id { get; set; }
+        public string Type { get; set; }
```

Comment:
how about making this an enum? It'd be a lot easier for customers to use it.

This is how message request type is defined in our OpenAPI spec:
```
        type:
          description: Contains type of the message
          enum:
            - regular
            - ephemeral
            - error
            - reply
            - system
            - deleted
```
and here's a C# definition I use in Unity SDK (generated by OpenAPI generator):
```
    public enum MessageRequestType
    {

        [System.Runtime.Serialization.EnumMember(Value = @"regular")]
        Regular = 0,

        [System.Runtime.Serialization.EnumMember(Value = @"ephemeral")]
        Ephemeral = 1,

        [System.Runtime.Serialization.EnumMember(Value = @"error")]
        Error = 2,

        [System.Runtime.Serialization.EnumMember(Value = @"reply")]
        Reply = 3,

        [System.Runtime.Serialization.EnumMember(Value = @"system")]
        System = 4,

        [System.Runtime.Serialization.EnumMember(Value = @"deleted")]
        Deleted = 5,

    }
```
this way it's less error-prone and devs can easily see what options there are